dashboards-reports/public/components/report_definitions/delivery/delivery.tsx

```diff
@@ -110,53 +119,92 @@ export function ReportDelivery(props: ReportDeliveryProps) {
   }
 
   const defaultCreateDeliveryParams = () => {
+    includeDelivery = false;
     reportDefinitionRequest.delivery = {
       configIds: [],
-      title: '',
-      textDescription: '',
+      title: `\u2014`, // default values before any Notifications settings are configured
+      textDescription: `\u2014`,
       htmlDescription: ''
     };
   };
 
-  const sendTestNotificationsMessage = () => {
-    // implement send message
+  const sendTestNotificationsMessage = async () => {
     // on success, set test message confirmation message
-    handleTestMessageConfirmation(testMessageConfirmationMessage)
+    // for each config ID in the current channels list
+    for (let i = 0; i < selectedChannels.length; ++i) {
+      try {
+        const eventId = await httpClientProps
+          .get(`../api/reporting_notifications/test_message/${selectedChannels[i].id}`, 
+          {
+            query: {
+              feature: 'reports'
+            }
+          })
+          .then((response) => response.event_id);
+          
+        await httpClientProps
+          .get(`../api/reporting_notifications/get_event/${eventId}`)
+          .then((response) => {
+            if (response.event_list.length != 1 && response.event_list[0].event_id != eventId) {
```

Comment:
when will it hit this if condition?

Reply:
It should hit this condition if the webhook URL is invalid or if sending the test message fails for whatever reason.

Reply:
send message failure will stil save the failure info in the event index with an event id. It's just the `delivery_status` will have statuscode as non-2xx, and with a statusText